backend/app/finders/spree/admin/reports/orders/top_products_by_line_item_totals.rb

```diff
@@ -13,13 +13,22 @@ def call
             variants = by_completed_at_min(variants)
             variants = by_completed_at_max(variants)
 
-            variants = variants.group('spree_variants.sku')
-                               .select('spree_variants.sku, sum(spree_line_items.quantity * spree_prices.amount) as line_item_total')
+            variants = variants.group(
+                                 'spree_variants.id',
```

Comment:
```suggestion
                                 "#{Spree::Variant.table_name}.id",
```

please don't hardcode table names in SQL queries